src/components/MainView.js

```diff
@@ -119,20 +119,15 @@ function MainView({ children }) {
               <footer
                 css={`
                   flex-shrink: 0;
-                  margin-bottom: ${5 * GU}px;
+                  padding-bottom: ${5 * GU}px;
                   width: 100%;
                   text-align: center;
                   ${textStyle('body2')};
                   color: ${theme.surfaceContentSecondary};
                 `}
               >
                 Supporting services provided by{' '}
-                <Link href="https://aragon.one">1Hive</Link>. Read 1Hive
-                Celeste's{' '}
-                <Link href="https://anj.aragon.org/legal/terms-general.pdf">
-                  Terms of Service
-                </Link>
-                .
+                <Link href="https://wiki.1hive.org">1Hive</Link>.
```

Comment:
should we re direct to the wiki here? or the discord?

Reply:
@willjgriff suggested to direct to the wiki. I agree it should be the wiki as well. 
There's a Discord link ion the wiki if users are looking to join Discord